playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts

```diff
@@ -7,14 +7,16 @@ import MyDashboardPage from 'pages/singular/dashboard/my-dashboard-page';
 import EnrollMyAdultDependentPage from 'pages/singular/enrollment/enroll-my-adult-dependent-page';
 import { WHO } from 'data/constants';
 
+const { SINGULAR_USER_EMAIL, SINGULAR_USER_PASSWORD } = process.env;
+
 async function signUp(page: Page) {
   await signMeUp(page);
 
   const preScreeningPage = new PreScreeningPage(page);
   await preScreeningPage.enterInformationAboutYourself();
 
   // Enter email alias and new password in Login popup
-  await auth.createAccountWithEmailAlias(page);
+  await auth.createAccountWithEmailAlias(page, { email: SINGULAR_USER_EMAIL, password: SINGULAR_USER_PASSWORD });
```

Comment:
`email` and `password` parameters are no longer optional. Both are required to have values explicitly set.
This change hopes to avoid confusion when we add other studies tests.